store/postgres.go

```diff
@@ -282,6 +283,64 @@ func (s *PGXStore) UpdateActor(ctx context.Context, opts UpdateActorOpts) (out *
 	return actor, nil
 }
 
+type UpdateActorProfileOpts struct {
+	// DID is the DID of the actor to update.
+	DID         string
+	ID          string
+	UpdatedAt   time.Time
+	IndexedAt   time.Time
+	DisplayName string
+	Description string
+}
+
+func (s *PGXStore) UpdateActorProfile(ctx context.Context, opts UpdateActorProfileOpts) (err error) {
```

Comment:
I'm torn on if the verb here should be `Update` or `Create`. Perhaps `CreateActorProfileUpdate` is a little more verbose but more accurately describes what's occurring in the database. 

I think there's a larger question here that's driven by the design further up - are we going to obscure the fact that actor_profiles are a separate entity and when RPCs like `GetActor` are called, are we going to always fetch the `candidate_actor` and latest `actor_profile`?